[14.0] [MIG] website_sale_stock_list_preview: Migration to 14.0 (#640)
OCA/e-commerce: website_sale_stock_list_preview/readme/DESCRIPTION.rst

> This module extends the functionality of e-Commerce to give
> the buyer the information about the stock on the preview of the products.

Reviewer: ```suggestion
This module extends e-Commerce's functionality to give the buyer information about what's in stock when previewing the products
```
